person.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..99_999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def can_use_services?
+    return true if @age >= 18 || parent_permission == true
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(_date, _book)
+    @rentals.push(rental)
+    @rental.person = self
```

Comment:
- Hey, this could be easy if you just instantiated here to avoid the added line like this 
` Rental.new(date, self, book)`